modules/understanding-upgrade-channels.adoc

```diff
@@ -24,6 +24,7 @@ binary file for a specific version of {product-title} always installs that versi
 * `candidate-{product-version}`
 * `fast-{product-version}`
 * `stable-{product-version}`
+* `eus-4.6` (only available when running 4.6)
```

Comment:
Is this channel available now? I did not see it from a v4.6.6 cluster. But following date in [1] says it's available on October 27, 2020.
```
Version	General availability	Full support ends	Maintenance support ends
4.6 EUS	October 27, 2020	Release of 4.7 + 1 month	Release of 4.7 + 14 months
```
[1] https://access.redhat.com/support/policy/updates/openshift#ocp4_phases

Reply:
Not yet, we'll land these documentation changes and the new channel at the same time.

Reply:
ok, then lgtm.

Reply:
> Not yet, we'll land these documentation changes and the new channel at roughly the same time as we ship 4.6.7, that's the plan at least.

@sdodson, do you want me merge this change after the 4.6.7. release notes go out?

Reply:
I'm not too concerned about the ordering as long as it's roughly around the same time.

Reply:
@sdodson the release notes for 4.6.8 (including the callout for eus-4.6) were published this morning and are now available: https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-8-eus-channel. Should we proceed in merging this now?

Reply:
yes please! the channel is now live as well.